src/gui/MainWindow.cpp

```diff
@@ -308,12 +308,13 @@ void MainWindow::finalize()
 					SLOT( exportProjectTracks() ),
 					Qt::CTRL + Qt::SHIFT + Qt::Key_E );
 
-	// temporarily disabled broken MIDI export				
-	/*project_menu->addAction( embed::getIconPixmap( "midi_file" ),
+#ifdef LMMS_HAVE_FLUIDSYNTH
```

Comment:
I agree with @softrabbit.  The days of MIDI = cheesy instrument bank are in the past but current version warns when no default soundfont is loaded, so perhaps decoupling this should be a separate bug/pr?